schema/project-level/codelists/partyRole.csv

```diff
@@ -11,3 +11,4 @@ reviewBody,Review body,A party responsible for the review of this procurement pr
 interestedParty,Interested party,"A party that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",OCDS
 publicAuthority,Public Authority,The entity responsible for developing the infrastructure assets and/or delivering the public services in this project.,OC4IDS
 administrativeEntity,Administrative Entity,The entity responsible for contract administration.,OC4IDS
+climateFinanceFocalPoint,Climate finance focal point,An entity that serves as the interface between the government of the country in which the infrastructure project is located and climate finance funders. The focal point is responsible for identifying and/or endorsing project ideas prior to submission to the climate finance funder.,OC4IDS
```

Comment:
Plural/singular don't agree, but can change to indefinite article to make it work.

```suggestion
climateFinanceFocalPoint,Climate finance focal point,An entity that serves as the interface between the government of the country in which the infrastructure project is located and climate finance funders. The focal point is responsible for identifying and/or endorsing project ideas prior to submission to a climate finance funder.,OC4IDS
```